plugins/lsp/language-servers/default.nix

```
@@ -20,6 +20,11 @@ with lib; let
       description = "Enable bashls, for bash.";
       package = pkgs.nodePackages.bash-language-server;
     }
+    {
+      name = "biome";
+      description = "Enable Biome, Toolchain of the Web";
+      package = pkgs.biome;
+    }
```

```suggestion
    {
      name = "biome";
      description = "Enable Biome, Toolchain of the Web";
    }
```
Actually we don't need this line either because by default, we assume that the corresponding package has the same name as the language server.

---

ah! makes sense. I was wondering why some didn't have a `package`